core/src/index/cmake/BuildUtilsCore.cmake

```diff
@@ -202,3 +202,17 @@ function(ADD_THIRDPARTY_LIB LIB_NAME)
         message(FATAL_ERROR "No static or shared library provided for ${LIB_NAME}")
     endif ()
 endfunction()
+
+MACRO(using_ccache_if_defined MILVUS_USE_CCACHE)
+    if (MILVUS_USE_CCACHE)
+        find_program(CCACHE_FOUND ccache)
+        if (CCACHE_FOUND)
+            message(STATUS "Using ccache: ${CCACHE_FOUND}")
+            set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${CCACHE_FOUND})
+            set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ${CCACHE_FOUND})
```

Comment:
since "ccache" can be found, need not use full path name
            set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
            set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)

Reply:
Use full paths is clearer for the compilation process